Source/WebCore/rendering/RenderCombineText.h

```diff
@@ -53,6 +54,7 @@ class RenderCombineText final : public RenderText {
     float m_combinedTextWidth { 0 };
     float m_combinedTextAscent { 0 };
     float m_combinedTextDescent { 0 };
+    String m_renderedText;
```

Comment:
I am not even sure if I agree with this approach, but since RenderCombine is going to go through a proper refactoring soonish, I guess this will do for now.